domains/Contact/ManageContactAddresses/Jobs/FetchAddressGeocoding.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Contact\ManageContactAddresses\Jobs;
+
+use App\Contact\ManageContactAddresses\Services\GetGPSCoordinate;
+use App\Models\Address;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+
+class FetchAddressGeocoding implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    /**
+     * The address instance.
+     *
+     * @var Address
+     */
+    public Address $address;
```

Comment:
Are you sure serialize this model in the job queue is working?

Reply:
Yes! And also, why wouldn't it work?